SimplyPluralApi.yml

```diff
@@ -1858,69 +1964,92 @@ paths:
       responses:
         "200":
           description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "./SimplyPluralApiBodies.yml#/components/schemas/chatChannelContainer"
         "404":
           description: NOT FOUND
 
-  /v1/board/:id:
+  # TODO: The below should exist but doesn't work?
+  /v1/board:
```

Comment:
Don't create seperate endpoints for one and the same endpoint with an optional id.